### DISCUSSION TITLE
CONTROLLARE UN INTERRUTTORE CON INTERRUZIONE DI NEUTRO

### DISCUSSION THREAD

**Discussion by s-dimaio** [indomus/forum#2920] [ZigBee, Z-Wave, Thread, Matter]
Buongiorno a tutti, vorrei poter controllare da remoto un interruttore presente all'interno di un armadio (accende dei faretti incastonati nel cartongesso sopra l'armadio) come già faccio con tutti gli altri interruttori della casa. Mi sono però accorto che in questo caso il falegname non ha interrotto la fase ma il neutro. In cassetta ho quindi il neutro che poi va alla lampada ed ho la possibilità di portare una fase da una presa vicina ma non ho la possibilità di avere la fase che va ai faretti nel cartongesso.

Esiste un qualche switch smart che funziona in una situazione come quella descritta? L'ideale sarebbe poter utilizzare uno switch sonoff (nel resto della casa ho sonoff zbmini e zbextreme ma sempre con interruzione di fase) ma in alternativa valuto anche altre soluzioni.

 
Grazie in anticipo.

 
Ciao,

 
Simone

**Comment by KC-inDomus** [comment]
purtroppo no
